src/vulkan/vulkan_renderer.h

```diff
@@ -37,6 +38,7 @@ namespace IC {
         PipelineManager _pipelineManager{};
         DescriptorAllocator _meshDescriptorAllocator{};
         DescriptorAllocator _imGuiDescriptorAllocator{};
+        VulkanTextureManager _textureManager{_vulkanDevice};
```

Comment:
Can you do this in the constructor's initilizer list instead? I think having all non-default initializations in one place makes it easier to read and debug in the future.

Reply:
I sort of disagree with this. imo I don't want to clutter the constructor's initializer list with things that don't need information passed into the constructor. I'm not gonna fight it too hard if you really think it should be there but if we add 4, 5, 10? more objects that need to be initialized there it's gonna be a mess right?